monitors/index.js

```diff
@@ -28,22 +30,32 @@ const ExpandedERC20 = artifacts.require("ExpandedERC20");
  * @param {String} address Contract address of the EMP.
  * @return None or throws an Error.
  */
-async function run(price, address, shouldPoll, botMonitorObject, walletMonitorObject, pollingDelay) {
+async function run(price, address, shouldPoll, botMonitorObject, walletMonitorObject, pollingDelay, priceFeedConfig) {
   Logger.info({
     at: "Monitor#index",
     message: "Monitor started 🕵️‍♂️",
     empAddress: address,
     currentPrice: price,
-    pollingDelay: pollingDelay
+    pollingDelay: pollingDelay,
+    priceFeedConfig
   });
 
   // Setup web3 accounts an contract instance
   const accounts = await web3.eth.getAccounts();
   const emp = await ExpiringMultiParty.at(address);
 
+  // Setup price feed.
+  // TODO: consider making getTime async and using contract time.
+  const getTime = () => Math.round(new Date().getTime() / 1000);
```

Comment:
yup. you can get the contract time easily from the method that is already there using `getLastUpdateTime`